### ISSUE TITLE
Fatal Error on start with Polkadot node running - Unhandled Rejection (TypeError): api.query.sudo is undefined

### ISSUE THREAD

**Issue by NukeManDan** [substrate-developer-hub/substrate-front-end-template#144]
I have a node template freshly compiled and running (2.0.0 using the 10-6 nightly toolchain) on my machine

(Note this happens with or without the node running!)

I now follow the tutorials for install with depth:

```bash
git clone -b v2.0.0 --depth 1 https://github.com/substrate-developer-hub/substrate-front-end-template
cd substrate-front-end-template
yarn install
yarn start
```

And see the UI for a moment with nonsense numbers for `blockNumber`, followed by a string of related errors on the console: 

![image](https://user-images.githubusercontent.com/35669742/96311363-87219b80-0fc6-11eb-9430-cb7fb6d3db67.png)

Source: 
src/substrate-lib/components/TxButton.js:39

**Comment by NukeManDan** [comment]
AHH I have a polkadot node running, that is what the template is trying to hook to, the node can't bind to the 9944 defalut port

**Comment by NukeManDan** [comment]
So this makes sense - sudo keys are not available ;) 
But I would _hope_ that the template could be used by those without sudo keys on any substrate based node. So I think the issue still remains to be fixed.

**Comment by danforbes** [comment]
Closed by #146
